frontend/src/components/explorer/gemBrowser/Metabolite.vue

```diff
@@ -75,7 +75,7 @@
                   {{ metabolite.name }} via ChEBI</a>
               </a>
             </div>
-            <div class="column is-3-widescreen is-3-desktop is-full-tablet has-text-centered">
+            <div class="column is-narrow">
```

Comment:
Since the new maps available card does not adapt to full-width for tablets, the spacing looks a bit weird for metabolites that have a chebi image. Not sure if this should be solved in this PR or a new issue.

<img width="1003" alt="image" src="https://user-images.githubusercontent.com/423498/122762736-e8a8b100-d29d-11eb-82e4-4394b5481b27.png">

Reply:
I'd say we should try to avoid `is-narrow` by controlling the width with the number of columns. There was some time spent on deciding these for the different widths, so please have an extra look that all the pages look good when going through the different widths.

Reply:
Not sure what you mean by "at source" but the reason the card contents is not filling up the card in the screenshot you provided is because of the table layout used inside the maps available component. It has nothing to do with having `is-narrow` vs `is-full-tablet` on the wrapper column.

Reply:
To add to this, what `is-narrow` does now is that it makes sure the maps available component is at 100% width and adapts the remaining column/s accordingly to fill up the remaining space.

Reply:
By "at source" I mean deciding on the card width in the source file where the component is defined, ie https://github.com/MetabolicAtlas/MetabolicAtlas/blob/2e7164739737e0e9281f57c4f16d58064f2f3c8c/frontend/src/components/explorer/gemBrowser/MapsAvailable.vue#L96-L98

Instead, it would be more reusable to decide the width in the parent component that uses it.

> makes sure the maps available component is at 100% width 

@e0 this is exactly the "inconsistent layout" that I think should be avoided, as some of the subsystem maps have really long names.

Reply:
It's great that you caught this! I think it might be better to focus on responsiveness with PR  #632 since all components will share the same file for layout. Since this is currently looking good for desktop, and that we will most likely close PR #632 within this sprint, would it be ok if I resolve this conversation and work with mobile and tablet responsiveness in that PR instead?

Reply:
> By "at source" I mean deciding on the card width in the source file where the component is defined, ie
> 
> https://github.com/MetabolicAtlas/MetabolicAtlas/blob/2e7164739737e0e9281f57c4f16d58064f2f3c8c/frontend/src/components/explorer/gemBrowser/MapsAvailable.vue#L96-L98
> 
> Instead, it would be more reusable to decide the width in the parent component that uses it.
> 
> > makes sure the maps available component is at 100% width
> 
> @e0 this is exactly the "inconsistent layout" that I think should be avoided, as some of the subsystem maps have really long names.

Ah ok, I think this is a good point about not having a `max-width` and it's worth rethinking how to handle that. The problem with using something like `is-3-desktop` vs `is-narrow` is that it sets a strict width for whatever is inside the column so it's difficult to work out how to layout the contents and would not play well with what we have now. I just wanted to clarify on for the cases where the component names are short, the table would still not fill up 100% of the column and that it is not due to `is-narrow`.

Reply:
@inghylt sure! In that case, please update #632 and/or #617 to mention that they should address the responsiveness as well.